tools/CMakeLists.txt

```diff
@@ -1,6 +1,11 @@
+option(INSTALL_EXAMPLES "" ON)
```

Comment:
Is it worth adding a description to the option that describes what exactly is installed?

From a very high level all the .bt files themselves also are examples, the whole purpose of bpftrace is to have hackable scripts, so it's not really clear that examples means just the script outputs here.

Reply:
> Is it worth adding a description to the option that describes what exactly is installed?

Obviously, just forgot to fill it in eventually...

Will fix that once I can.

> From a very high level all the .bt files themselves also are examples, the whole purpose of bpftrace is to have hackable scripts, so it's not really clear that examples means just the script outputs here.

Well yes, they are all example usages for the same program, but I used the term to refer their ouput as shown in the .txt files, because these are examples... of output/results.

I'm all in for a better word if someone comes up with it.

Reply:
At the risk of bike shedding, how about something like `INSTALL_TOOL_DOCS` ?

Reply:
Sure, why not.
Will update in a bit.